### PR TITLE
Added Joao's Part

### PR DESCRIPTION
**Pull request by Johnny-ESA**
Is this ok?

Also, do we wanna make the Headers with our names `section`s instead of `subsection`s?

### CODE REVIEW

File: Shared.tex [hunk 1]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
 \subsection*{Helene Pfahler}
 I contributed to this project by working on the Aitviewer as well. Additionally, I wrote reports for our weekly team meetings.
 
-\subsubsection*{... to be continued ...}  
+\subsubsection*{... to be continued ...}
+
+
+\subsection*{Joao Espirito Santo Azevedo}
+\subsubsection*{Introduction}
+In this Section i aim to report my contributions to this teamproject.\\
```
Comment (helepf) [inline Shared.tex]: Orthography: 
- I instead of i 
- team project instead of teamproject

File: Shared.tex [hunk 2]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+As hinted to in the previous Section, to work on this Teamproject, we chose to split up our Group into two: one part would be responsible for setting up/handling the AI model on the cluster, while the other would be responsible for handling the UI/the AItviewer Application.
```
Comment (helepf) [inline Shared.tex]: This might be unnecessary, because it is already included in the "about this project" section

File: Shared.tex [hunk 3]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+
+\subsubsection*{Personal difficulties}
+\quad For the first few weeks of this Project i was involved in the Model-Group, trying to set up the Cluster on which the AI should run.
+
+\quad I soon found this task was somewhat outside my area of expertise and chose to switch over to the UI team accordingly. Since Jan joined the Model team simultaneously and the UI Team was quite helpful in $"$showing me the ropes$"$ of the whole Aitviewer-Programme and -Code, i believe this transition was able to be handled significantly well.\\
```
Comment (helepf) [inline Shared.tex]: The indentations are nice, but to keep the layout consistent throughout the whole document, we would need to add them either everywhere or nowhere. For the same reason, I would suggest using the term "sub-team" consistently (but of course if you have another suggestion for this term, let me know it :))

Also, for orthography and a little more formal style of writing, my suggestion would be:

For the first few weeks of this project I was involved in the model sub-group, participating in the setup of the cluster on which the AI runs.
I soon found this task to be somewhat outside my area of expertise and chose to switch over to the UI sub-group accordingly. Thankfully, this transition went smoothly, because Jan joined the model sub-group instead and I could quickly be introduced to the functioning of the Aitviewer programme.

File: Shared.tex [hunk 4]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+
+\quad A further difficulty i had in the course of this project were two weeks where my laptop had technical problems with its fans and i was forced to try and relocate all my university work and such onto my older, significantly slower, Laptop.
+
+\subsubsection*{Filename-Input Loader}
+\subsubsection*{--Context}
+\quad In the beginning of this project, if we wanted to load a specific Sequence/file with an animation onto the AItviewer, we had to change the corresponding String in the `load\_AMASS.py`-file. After a few weeks in the Model group, it became apparent that this was an annoying and inconvenient task/method.
+
```
Comment (helepf) [inline Shared.tex]: I find the term sequence to be a little bit confusing in this context. Maybe just use file :)

Also, be careful with capitilization... I, model, laptop, loader, ...

File: Shared.tex [hunk 5]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+\quad After some planning/brainstorming among us i assumed the task to handle the creation of a similar python-file (a Loader, if you will), which allowed the user to specify which file should be loaded into the AItviewer at the time of executing the file.
+
+\subsubsection*{--how it works}
```
Comment (helepf) [inline Shared.tex:137]: Very good idea to split a topic into context, "how it works" and "possible future problems" :)

Maybe, in our favor, we could just not mention the possible problems for the future and simply pretend we did an awesome job haha

File: Shared.tex [hunk 6]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+\quad The file i worked on, `load\_export\_AMASS.py` relies heavily on the os-library of python.
+To use this Loader, it suffices to call the command `python load\_export\_AMASS.py` in the WSL/Linux Terminal while inside the `examples`-directory of the github repository.
+
+\todo[color=grey!8, inline, bordercolor=white]{This command should work in any directory with a serving python-environment, though, so long as the `load\_export\_AMASS.py`-part of it is expanded with the corresponding relative path to it. (i.e. `../examples/load\_export\_AMASS.py`).}
```
Comment (helepf) [inline Shared.tex:141]: The [color=grey!8, inline, bordercolor=white] doesn't work yet, but I know you said this documentary is still in progress. I also think our document looks the cleanest if the only font color is black, so maybe you don't need it.

File: Shared.tex [hunk 7]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+
+\quad After a little while, the Loader should present an indexed list of the contents within the `export`-folder of the local repository. At the same time, the Loader should prompt the user to input a number specifying which of the files in this folder should be opened. Upon typing the number and pressing `Enter`, the Loader should "do its magic".
```
Comment (helepf) [inline Shared.tex]: I know you tried your very best to make this text sound as formal as possible. Maybe change "do its magic" to "open the selected file".

File: Shared.tex [hunk 8]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+
+\subsubsection*{--Possible future problems}
+If one desires to work on this Loader, possible problems that they might encounter might be due to:\\
```
Comment (helepf) [inline Shared.tex]: No enumerations please! Chuqiao requested this to be a continuous text

File: Shared.tex [hunk 9]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+- Some paths not being relative(in relation to the current directory upon executing the Loader) or absolute(the complete path starting from the user's root-directory)\\
+- Missing/incorrect permissions of certain folders (if the user is not permitted to open a folder, it probably won't open, even with the Loader)\\
+- Incorrect Formats of the files to open (if the files are not stored in a format which the AItviewer is prepared to handle/understand, it won't work). - This problem is not fixable in the Loader, but rather probably in the `SMPLSequence.py`-file\\
+- Problems with indexing/handling the List of directories (Double-check how these are handled)\\
+
+\subsubsection*{Example Inputs}
+As mentioned before, the UI Team worked together to create a couple animations to use as corner-case tests for the software.\\
```
Comment (helepf) [inline Shared.tex]: ... as corner-case tests for the model?

Let's just write these as sentences, not as a list

File: Shared.tex [hunk 10]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+The ones i created were all based on the `Female1 Walking` Animation:\\
+- One was multiple frames (ca. 15), spread evenly throughout the whole animation\\
+- One had no changes\\
+- And one had changes applied to only the 10th first and the 10th last frames\\
+
+\subsubsection*{Prompts}
+\subsubsection*{--Context}
+Many of the animations we were given to open in the AItviewer included prompts, which described the actions performed by the character and other information. Our task was to incorporate a way to view these in the Aitviewer.
```
Comment (helepf) [inline Shared.tex]: Orthography: Aitviewer instead of AItviewer

File: Shared.tex [hunk 11]
```diff
@@ -102,7 +102,57 @@ \subsubsection*{Prompts and their Export}
...
+
+\subsubsection*{--How it works}
+The prompts are given as a multidimensional array. Every entry in its first dimension is a Prompt of the animation. These, in turn, are also arrays.\\
+To view these, we expanded the AItviewer Interface a bit further in the `SMPLSequence.py`-file by creating a respective interface therein. One can interact with it on the AItviewer by going onto the `Editor`-window, clicking on the `AMASS Running`-dropdown menu and then on its `Prompts`-Button.\\
+This Interface allows the user to individually view only the current Prompts as well as all Prompts. Additionally, the user is welcome to add or delete Prompts, and specify their Timespans.
```
Comment (helepf) [inline Shared.tex]: I wouldn't capitalize the word "prompts". 

But nice paragraph!

**Review by helepf — COMMENTED** [review]
Good work! Just a few hints:
- double-check the orthography, especially typos like AItviewer
- no indents please
- no lists please

**Comment by helepf** [comment]
Oh and please add your full name and Matrikelnummer into the header :)

File: Shared.tex (hunk 2 above)
Comment (Johnny-ESA) [reply Shared.tex]: Oh! Didn't see that part, sorry...

File: Shared.tex (hunk 3 above)
Comment (Johnny-ESA) [reply Shared.tex]: Grand! I'll try to sweep out the "team"/"group"s for "sub-group", then

Also, thank you lots for the paragraph-suggestion!! I'll largely incorporate it into my fix

File: Shared.tex (hunk 5 above)
Comment (Johnny-ESA) [reply Shared.tex:137]: Well, I feel I have to respectfully disagree there...:

I feel as though it's to some extent a better idea to incorporate little hints that might help future programmers debug their own code...

But if you insist, I can delete this, please let me know

File: Shared.tex (hunk 6 above)
Comment (Johnny-ESA) [reply Shared.tex:141]: Oh, that's not the font-color: i merely wanted to add a small comment-like snippet here, since i felt this feature was something i'm rather proud of, but not that fitting in the flow of the text...

It not working is because i forgot the `\usepackage{todonotes}` in the top, oops!

**Review by helepf — APPROVED** [review]
Supidupi
